### ISSUE TITLE
Feature Request > Cart is persistent across sessions for logged in users

### ISSUE THREAD

**Issue by peytonchance** [craftcms/commerce#487]
### Description
In working on an in-progress project, I've noticed that if a logged-in user gathers items into their cart and then logs out, upon logging back in, the cart is empty. The user then has to re-gather all of the items in order to check out.

The hope would be that a registered user could use the same cart across sessions without losing the items they have gathered. The cart would be cleared only once the user has submitted the order (or removed all items from the cart) and not between sessions.

**Comment by lukeholder** [comment]
@peytonchance thanks for the feature request. Will look into this.

**Comment by rob-baker-ar** [comment]
This would be great to have.

**Comment by lukeholder** [comment]
Duplicate of #704
